newdle/client/src/components/NewdlesImIn.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {Container, Icon, Label, Placeholder} from 'semantic-ui-react';
+import {useHistory} from 'react-router';
+import {serializeDate, toMoment} from '../util/date';
+import client from '../client';
+import {usePageTitle} from '../util/hooks';
+import styles from './MyNewdles.module.scss';
+
+export default function NewdlesImIn() {
+  const [newdles, loading] = client.useBackend(() => client.getNewdlesImIn(), []);
+  usePageTitle("Newdles I'm in");
+
+  let content;
+  if (loading || newdles === null) {
+    content = (
+      <>
+        <Placeholder className={styles.newdle} />
+        <Placeholder className={styles.newdle} />
+        <Placeholder className={styles.newdle} />
+      </>
+    );
+  } else if (newdles.length === 0) {
+    content = (
+      <div className={styles['no-newdle-container']}>
+        {/* eslint-disable-next-line jsx-a11y/accessible-emoji */}
+        <h2>You are not part of any newdles yet... 🍜</h2>
+      </div>
+    );
+  } else {
+    content = newdles.map(newdle => <NewdleAsParticipant key={newdle.id} newdle={newdle} />);
+  }
+
+  return (
+    <Container text className={styles.newdles}>
+      {content}
+    </Container>
+  );
+}
+
+function NewdleAsParticipant({newdle: {code, title, duration, final_dt: finalDT, timezone}}) {
+  const history = useHistory();
+  const startTime = finalDT ? serializeDate(finalDT, 'HH:mm') : undefined;
```

Comment:
```js
const startTime = finalDT && serializeDate(finalDT, 'HH:mm');
```
is less explicitly but more idiomatic (and does the same thing in this case). Nothing wrong about the way you're doing it though.

Reply:
OK, @ThiefMaster has just pointed out to me that this is probably copy-paste from the other page, so it's something which was already there (and somehow managed to evade the code review). So, never mind, we'll fix both later.

Reply:
I agree, I prefer your suggestion, I've been writing most code in the same way. I didn't cover this case exactly by the same reason as the second comment, keep both in the same format. But I can start by adding this change here.